server/views/gtm-datalayer/index.njk

```diff
@@ -7,8 +7,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <!-- Bootstrap includes for asu-web-standards-bootstrap4-->
   <link rel="stylesheet" href="../@asu/unity-bootstrap-theme/css/unity-bootstrap-theme.bundle.css" type="text/css" media="all">
-  <script type="text/javascript" src="./@asu/unity-bootstrap-theme/js/bootstrap.bundle.min.js"></script>
-  <script type="text/javascript" src="./@asu/unity-bootstrap-theme/js/global-header-datalayer.js"></script>j
+  <script type="text/javascript" src="../@asu/unity-bootstrap-theme/js/bootstrap.bundle.min.js"></script>
+  <script type="text/javascript" src="../dataLayerExample.js"></script>
```

Comment:
Note - We need the same thing done for the dataLayerExample in the asuheader page.

Also, for some reason, I wasn't seeing this in my local testing. It was still using the old `global-hader-datalayer.js`. Not sure if it was user error or of something else is amiss.